### DISCUSSION TITLE
Pieces for Developers Linux app always says disconnected from the internet

### DISCUSSION THREAD

**Discussion by rlperez** [pieces-app/support#137] [Q&A]
I am not terribly familiar with snaps work but my computer is Fedora 39 Gnome/Wayland. This is my snap interfaces so far. 

```
❯ snap interfaces | grep pieces
gnome-42-2204:gnome-42-2204                       pieces-for-developers,pieces-os
gtk-common-themes:gtk-3-themes                    pieces-for-developers,pieces-os
gtk-common-themes:icon-themes                     pieces-for-developers,pieces-os
gtk-common-themes:sound-themes                    pieces-for-developers,pieces-os
pieces-for-developers:dbus-pieces-for-developers  -
pieces-os:dbus-pieces-os                          -
:desktop                                          pieces-for-developers,pieces-os
:desktop-legacy                                   pieces-for-developers,pieces-os
:gsettings                                        pieces-for-developers,pieces-os
:home                                             pieces-for-developers,pieces-os
:network                                          pieces-for-developers,pieces-os
:opengl                                           pieces-for-developers,pieces-os
:process-control                                  pieces-os
:wayland                                          pieces-for-developers,pieces-os
:x11                                              pieces-for-developers,pieces-os
```

I am not too sure what the issue is. I don't see any weird behavior when using the plugin with VS code. Just this. I've tried it with my VPN on and off. Restarted. It also lets me connect to my Github account so it at least seems to have outward connectivity. If I change from Mistral 7B to a cloud provider like OpenAI it still answers my questions. I see there is an interface when I list interfaces with `snap interfaces` for network status. Does that need to be connected? I can't seem to make it function.

**Comment by caleb-at-pieces** [comment]
This might do better as an issue instead of a discussion.

I spoke with the team & they mentioned this is likely due to an issue on your internet router, as there could be a DNS filter or firewall that is blocking requests.

We are working on a fix for this so it should be fixed within the next release of the desktop app.
